src/ITensors.jl

```diff
@@ -182,3 +182,13 @@ end
 # TODO: make this definition AbstractMPS
 # Handle orthogonality center correctly
 Base.getindex(ψ::MPS, r::UnitRange{Int}) = MPS([ψ[n] for n in r])
+
+#TODO Remove if everything is working nicely
+#Was still crashing on my laptop after updating ITensors
+Base.fill!(::NDTensors.NoData, ::Any) = NDTensors.NoData()
```

Comment:
I added a similar definition in https://github.com/ITensor/ITensors.jl/pull/1173, is the definition there not sufficient?